hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/CapacityScheduler.md

```diff
@@ -744,6 +744,14 @@ Changing queue/scheduler properties and adding/removing queues can be done in tw
 
   Remove the queue configurations from the file and run refresh as described above
 
+### Enabling periodic configuration refresh
+Enabling queue configuration periodic refresh allows changing the configuration by editing the *conf/capacity-scheduler.xml* without the necessicity of calling yarn rmadmin -refreshQueues.
+
+| Property | Description |
+|:---- |:---- |
+| `yarn.resourcemanager.scheduler.monitor.enable` | Enabling monitoring is necessary for the periodic refresh. Default value is false. |
+| `yarn.resourcemanager.scheduler.monitor.policies` | You should add `org.apache.hadoop.yarn.server.resourcemanager.capacity.QueueConfigurationAutoRefreshPolicy` to the policies list. Default value is `org.apache.hadoop.yarn.server.resourcemanager.monitor.capacity.ProportionalCapacityPreemptionPolicy`. |
```

Comment:
Please do not use "you should..." here, as @bteke suggested, please use gerunds.
Also, can you describe in a short sentence what's the difference between ProportionalCapacityPreemptionPolicy vs. QueueConfigurationAutoRefreshPolicy and why the user should use the latter? 
Thanks.

Reply:
Also, please also describe that this is a configuration property that holds a list of classes. So actually, it can be not too useful to remove the ProportionalCapacityPreemptionPolicy as it would turn off the preemption feature, so it should be explained in a nutshell that by adding more classes means more monitor tasks will be launched. Please make this a bit more disambiguous.